test/unit/ndr_error/log_test.rb

```diff
@@ -205,14 +205,9 @@ def setup
     end
 
     test 'should store the database name if configured to do so' do
-      begin
-        original_database_identifier = NdrError.database_identifier
-        NdrError.database_identifier = -> { 'SQLite test DB' }
-
+      with_config(:database_identifier, -> { 'SQLite test DB' }) do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.